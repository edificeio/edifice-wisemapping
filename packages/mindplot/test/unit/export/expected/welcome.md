# Welcome To WiseMapping

- 🖥️ 5 min tutorial video ?
Follow the link ! ( [link](https://www.youtube.com/tv?vq=medium#/watch?v=rKxZwNKs9cE) )
- 😮 Try it Now!
	- Double Click
	- Press "enter" to add a 
Sibling
	- Drag map to move
- Features
	- Links to Sites ( [link](http://www.digg.com) )
	- Styles
		- Fonts
		- Topic Shapes
		- Topic Color
	- 🌈 Icons
	- ⬅️ History Changes
- 👍 Mind Mapping
	- Share with Collegues
	- Online
	- Anyplace, Anytime
	- Free!!!
- 📊 Productivity
	- 💡 Share your ideas
	- Brainstorming
	- Visual 
- 🖥️ Install In Your Server
	- Open Source ( [link](http://www.wisemapping.org/) )
	- Download ( [link](http://www.wisemapping.com/inyourserver.html) )
- 👨‍👧‍👦 Collaborate
	- Embed
	- Publish
	- ✉️ Share for Edition

