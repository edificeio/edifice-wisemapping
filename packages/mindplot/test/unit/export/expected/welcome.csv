Welcome To WiseMapping 
,
            Try it Now! 
,
            ,
            Double Click 
,
            ,
             INS to insert 
,
            ,
            Drag map to move 
,
            Productivity 
,
            ,
            Share your ideas 
,
            ,
             
,
            ,
            Visual  
,
            Mind Mapping 
,
            ,
            Share with Collegues 
,
            ,
            Online 
,
            ,
            Anyplace, Anytime 
,
            ,
            Free!!! 
,
            Web 2.0 Tool 
,
            ,
            Collaborate 
,
            ,
            No plugin required 
,
            ,
            Share 
,
            ,
            Easy to use 
,
            Features 
,
            ,
            Links to Sites 
,
            ,
            Fonts 
,
            ,
            Topic Color 
,
            ,
            Topic Shapes 
,
            ,
            Icons 
,
            ,
            History Changes 
